apps/vth-frontend/src/styles/globals.css

```diff
@@ -1,3 +1,24 @@
+h1 strong,
+h2 strong,
+h3 strong,
+h4 strong,
+h5 strong,
+h6 strong {
+  font-weight: inherit;
```

Comment:
Er is toch een strong component in Utrecht met een design token? Kunnen we niet zorgen dat die werkt hier?